lib/rubocop/cop/rails/active_model_errors_direct_manipulation.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks direct manipulation of ActiveModel#errors as hash.
+      # These operations are deprecated in Rails 6.1 and will not work in Rails 7.
+      #
+      # @safety
+      #   This cop is unsafe because it can report `errors` manipulation on non-ActiveModel,
+      #   which is obviously valid.
+      #   The cop has no way of knowing whether a variable is an ActiveModel or not,
```

Comment:
Is the comma at the end of the sentence a typo?

Reply:
Yes, my bad. I've corrected this.